### DISCUSSION TITLE
Include more search paths for Proto

### DISCUSSION THREAD

**Discussion by dkdog** [w0rp/ale#4440] [Q&A - Ask for help with problems]
How do I pass on more search paths to protoc, ie. `protoc -I some_random_path` from **Ale**?

AFAICT, it's ALE that's showing me `XXX is not defined` because the `import "dir/dir/foo.proto"` statement in my proto file needs to be searched from another specific directory.

**Comment by gitsang** [comment]
I have encountered the same issue.

![image](https://github.com/dense-analysis/ale/assets/49085035/3d59daab-f950-4781-8046-d5b5e9eb6521)

I think this issue arises from an incorrect `proto_path` configuration. 
However, I haven't found a place to configure the `proto_path`.

### Using `protoc --lint_out`

```sh
➤  ~/.tmp/1688700945
 % protoc --proto_path=./third_party --proto_path=./api --lint_out=. ./api/helloworld/v1/*.proto
error_reason.proto: File not found.
helloworld/v1/greeter.proto:6:1: Import "error_reason.proto" was not found or had errors.
helloworld/v1/greeter.proto:31:3: "helloworld.v1.Error" seems to be defined in "helloworld/v1/error_reason.proto", which is not imported by "helloworld/v1/greeter.proto".  To use it here, please add the necessary import.

➤  ~/.tmp/1688700945
 % protoc --proto_path=./third_party --proto_path=./api/helloworld/v1 --lint_out=. ./api/helloworld/v1/*.proto
```
